vector/src/main/java/im/vector/app/features/location/live/CheckIfLiveLocationShareIsRedactedUseCase.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.location.live
+
+import org.matrix.android.sdk.api.session.Session
+import timber.log.Timber
+import javax.inject.Inject
+
+class CheckIfLiveLocationShareIsRedactedUseCase @Inject constructor(
```

Comment:
Yes you are right. I extracted the use case into a dedicated `feature/redaction` package so that it can be reused elsewhere. See [7252047](https://github.com/vector-im/element-android/pull/6486/commits/7252047605a7607afd17d9df6d7ba0b2a8704393)